src/common/com/intellij/plugins/haxe/build/MethodWrapper.java

```diff
@@ -17,48 +17,39 @@
  */
 package com.intellij.plugins.haxe.build;
 
+import org.jetbrains.annotations.NotNull;
+
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
 /**
  * Created by ebishton on 1/16/16.
  */
-public class ApiInvoker<R> {
+public class MethodWrapper<R> {
 
-  private Object myInstance;
   private Method myMethodToInvoke;
   private String myDebugName;
 
-  public ApiInvoker(Object o, String methodName, Class<?>... argTypes) {
-    Class klass;
+  public MethodWrapper(@NotNull Class klass, @NotNull String methodName, Class<?>... argTypes) {
```

Comment:
ApiInvoker got renamed to MethodWrapper.  It also throws an UnsupportedMethodException which was renamed from UnsupportedApiException.